SCons/Environment.py

```diff
@@ -1623,7 +1644,7 @@ def ParseConfig(self, command, function=None, unique=1):
         """
         if function is None:
             def parse_conf(env, cmd, unique=unique):
-                return env.MergeFlags(cmd, unique)
+                env.MergeFlags(cmd, unique)
```

Comment:
Why remove the return?

Reply:
it doesn't really have anything to return, but there's no harm in keeping the return statement either - MergeFlags does not return anything specific, it modifies env in place.